subsys/dfu/src/dfu_target.c

```diff
@@ -72,6 +72,9 @@ int dfu_target_init(int img_type, size_t file_size)
 	 * it left off.
 	 */
 	if (new_target == current_target) {
+		if (img_type == DFU_TARGET_IMAGE_TYPE_MODEM_DELTA) {
```

Comment:
I know the commit message explains it, but I would add a comment here as well why do we need to reinitialize modem dfu engine specifically. It's not that obvious when reading the code, especially given the comment above:
```
	 * Avoid re-initializing to ensure that the download can continue where
	 * it left off.
```

Reply:
Sure that's a great idea I'll add it

Reply:
Thank you